sqswatcher/sqswatcher.py

```diff
@@ -206,6 +208,29 @@ def pollQueue(scheduler, q, t, proxy_config):
 
         time.sleep(30)
 
+
+def getInstanceMappingFile(region, proxy_config):
+    s3 = boto3.resource('s3', config=proxy_config)
+
+    try:
+        if not os.path.exists('/opt/cfncluster/'):
+            os.makedirs('/opt/cfncluster/')
+    except OSError as ex:
+        if ex.errno == errno.EEXIST and os.path.exists('/opt/cfncluster/'):
```

Comment:
Is there the possibility of a race condition when checking/creating `/opt/cfncluster` dir? If it's not the case I would avoid this further check and raise the exception if creation fails